db/schema.rb

```diff
@@ -36,12 +36,31 @@
     t.index ["quiz_id"], name: "index_questions_on_quiz_id", using: :btree
   end
 
+  create_table "quizmasters", force: :cascade do |t|
```

Comment:
I don't know.. I got this merge message yesterday, could that be it? check the last commit..